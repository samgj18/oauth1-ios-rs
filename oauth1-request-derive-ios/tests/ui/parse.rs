#[derive(oauth1_request_ios::Request)]
struct Test {
    #[oauth1]
    meta_word: u8,

    #[oauth1 = ""]
    meta_name_value: u8,

    #[oauth1("")]
    meta_list_lit: u8,

    #[oauth1(word word)]
    redundant_word: u8,

    #[oauth1(word word word)]
    redundant_words: u8,

    #[oauth1(name = value value)]
    redundant_value: u8,

    #[oauth1(name = value value value)]
    redundant_values: u8,

    #[oauth1(name = )]
    missing_value: u8,

    #[oauth1(, meta)]
    empty_head: u8,

    #[oauth1(meta, , meta)]
    empty_mid: u8,

    #[oauth1(meta, ,)]
    empty_tail: u8,
}

fn main() {}
