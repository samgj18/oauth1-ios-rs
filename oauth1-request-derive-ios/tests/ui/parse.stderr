error: expected attribute arguments in parentheses: #[oauth1(...)]
 --> tests/ui/parse.rs:3:7
  |
3 |     #[oauth1]
  |       ^^^^^^

error: expected parentheses: #[oauth1(...)]
 --> tests/ui/parse.rs:6:14
  |
6 |     #[oauth1 = ""]
  |              ^

error: expected identifier
 --> tests/ui/parse.rs:9:14
  |
9 |     #[oauth1("")]
  |              ^^

error: expected `,`
  --> tests/ui/parse.rs:12:19
   |
12 |     #[oauth1(word word)]
   |                   ^^^^

error: expected `,`
  --> tests/ui/parse.rs:15:19
   |
15 |     #[oauth1(word word word)]
   |                   ^^^^

error: expected `,`
  --> tests/ui/parse.rs:18:27
   |
18 |     #[oauth1(name = value value)]
   |                           ^^^^^

error: expected `,`
  --> tests/ui/parse.rs:21:27
   |
21 |     #[oauth1(name = value value value)]
   |                           ^^^^^

error: unexpected end of input, expected an expression
  --> tests/ui/parse.rs:24:21
   |
24 |     #[oauth1(name = )]
   |                     ^

error: expected identifier
  --> tests/ui/parse.rs:27:14
   |
27 |     #[oauth1(, meta)]
   |              ^

error: expected identifier
  --> tests/ui/parse.rs:30:20
   |
30 |     #[oauth1(meta, , meta)]
   |                    ^

error: expected identifier
  --> tests/ui/parse.rs:33:20
   |
33 |     #[oauth1(meta, ,)]
   |                    ^
